tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -539,9 +539,10 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
+		[ "$ARCH" == "ppc64le" ] && [ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu"
```

Comment:
Hi @Amulyam24 , should this line be "[ "$ARCH" == "ppc64le" ] && ([ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu)"? I see lots of CI fail at this line.

Reply:
Thanks @jongwu , yes there's a small glitch in the line. Let me rectify and re-push.